pytorch_toolkit/action_recognition/README.md

```diff
@@ -191,4 +191,4 @@ mo.py --input_model resnet34_vtn.onnx --input_shape '[1,16,3,224,224]'
 
 ## Demo
 
-You can try your models after converting them to the OpenVINO™ format or a [pretrained model from OpenVINO™](https://docs.openvinotoolkit.org/latest/usergroup10.html) using the [demo application from OpenVINO™ toolkit](https://docs.openvinotoolkit.org/latest/_demos_python_demos_action_recognition_README.html)
+You can try your models after converting them to the OpenVINO™ format or a [pretrained model from OpenVINO™](https://docs.openvinotoolkit.org/latest/text_detection.html) using the [demo application from OpenVINO™ toolkit](https://docs.openvinotoolkit.org/latest/omz_demos_python_demos_action_recognition_README.html)
```

Comment:
@abhatikar It seems to be a confusion come from the previous version of these changes. I guess, the Action Recognition Models are required for the demo, and not Text Detection. So please change https://docs.openvinotoolkit.org/latest/text_detection.html  to the actual link to Action Recognition Models is https://docs.openvinotoolkit.org/latest/action_recognition_models.html. Thank you!

Reply:
Done